src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Xamarin.Android.Sdk.targets

```diff
@@ -0,0 +1,20 @@
+<!--
+***********************************************************************************************
+Xamarin.Android.Sdk.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+  created a backup copy.  Incorrect changes to this file will make it
+  impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (C) 2011-2016 Xamarin. All rights reserved.
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <XamarinAndroidSdkTargetsImported>true</XamarinAndroidSdkTargetsImported>
+  </PropertyGroup>
+
+  <!-- Allows providing Xamarin SDK-specific targets overrides -->
+	<Import Project="Xamarin.Sdk.targets" Condition="'$(XamarinSdkTargetsImported)' != 'true' And Exists('Xamarin.Sdk.targets')"/>
```

Comment:
Nope, it shouldn't. The IDEs can provide one as needed. If this repo provided it, the IDE VSIX wouldn't be able to install one on its own